control-plane/cmd/webhook-kafka/main.go

```diff
@@ -49,6 +49,7 @@ var types = map[schema.GroupVersionKind]resourcesemantics.GenericCRD{
 	eventingv1alpha1.SchemeGroupVersion.WithKind("KafkaSink"):    &eventingv1alpha1.KafkaSink{},
 	sourcesv1beta1.SchemeGroupVersion.WithKind("KafkaSource"):    &sourcesv1beta1.KafkaSource{},
 	messagingv1beta1.SchemeGroupVersion.WithKind("KafkaChannel"): &messagingv1beta1.KafkaChannel{},
+	eventingcorev1beta1.SchemeGroupVersion.WithKind("Broker"):    &eventingcorev1beta1.Broker{},
```

Comment:
why `v1beta1`?

Reply:
wait wait, we don't have `beta1` anymore

Reply:
Look: the import is just bad named: https://github.com/knative-sandbox/eventing-kafka-broker/blob/5887163fa2ae513db4a598278184ce7fc5538c84/control-plane/cmd/webhook-kafka/main.go#L35

Reply:
Also, isn't already here: https://github.com/knative/eventing/blob/main/cmd/webhook/main.go#L63 ?

Reply:
Thanks for the heads up. Fixed it now.